lib/Containers/MerkleTree.cpp

```diff
@@ -620,8 +620,8 @@ void MerkleTree<Hasher, BranchingBits>::insert(
   std::vector<std::uint64_t> sortedKeys = keys;
   std::sort(sortedKeys.begin(), sortedKeys.end());
 
-  std::uint64_t minKey = sortedKeys[0];
-  std::uint64_t maxKey = sortedKeys[sortedKeys.size() - 1];
```

Comment:
g++-13 complains with a potential out of bounds access here, which clearly is a false positive given the above checks for the size of `sortedKeys` / `keys`.